### ISSUE TITLE
NativeQuery datatype timestamp conversion issue.

### ISSUE THREAD

**Issue by whitingjr** [Hyperfoil/Horreum#61]
Using the Series tab in the UI Horreum is throwing an error.

```
Caused by: org.postgresql.util.PSQLException: ERROR: operator does not exist: timestamp without time zone >= bytea
  Hint: No operator matches the given name and argument types. You might need to add explicit type casts.
  Position: 129
                at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2553)
                at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2285)
                at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:323)
                at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:481)
                at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:401)
                at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:164)
                at org.postgresql.jdbc.PgPreparedStatement.executeQuery(PgPreparedStatement.java:114)
                at io.agroal.pool.wrapper.PreparedStatementWrapper.executeQuery(PreparedStatementWrapper.java:78)
                at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.extract(ResultSetReturnImpl.java:57)
                at org.hibernate.loader.Loader.getResultSet(Loader.java:2303)
                at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:2056)
                at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:2018)
                at org.hibernate.loader.Loader.doQuery(Loader.java:948)
                at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:349)
                at org.hibernate.loader.Loader.doList(Loader.java:2849)
                at org.hibernate.loader.Loader.doList(Loader.java:2831)
                at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2663)
                at org.hibernate.loader.Loader.list(Loader.java:2658)
                at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:338)
                at org.hibernate.internal.SessionImpl.listCustomQuery(SessionImpl.java:2141)
                at org.hibernate.internal.AbstractSharedSessionContract.list(AbstractSharedSessionContract.java:1172)
                at org.hibernate.query.internal.NativeQueryImpl.doList(NativeQueryImpl.java:176)
                at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1593)
                at org.hibernate.query.Query.getResultList(Query.java:165)
                at io.hyperfoil.tools.horreum.api.GrafanaService.query(GrafanaService.java:108)
```
 Looks like this parameter needs to be handled with more type casting.

**Comment by jesperpedersen** [comment]
Please, use real data types -- Types.BLOB (bytea) isn't ideal...

**Comment by whitingjr** [comment]
[This](https://github.com/whitingjr/Horreum/tree/handle-timestamp-type) might deal with the native query type.

**Comment by rvansa** [comment]
The problem was not using blob type in schema; due to parsing conversions the query was sent with `null` in the `range.from` and `range.to` and Postgresql interpretted that as a blob. Problem was fixed in b721dfce1c4858109a7a92e1485e1dab829fd1f8
